ban/commands/init.py

```diff
@@ -34,10 +35,17 @@ def init(*paths, limit=0, **kwargs):
             print('Computing file size')
             total = sum(1 for line in helpers.iter_file(path))
             print('Done computing file size')
-        helpers.batch(process_row, rows, chunksize=100, total=total)
+        helpers.batch(process_rows, rows, chunksize=100, total=total)
 
 
 @helpers.session
+def process_rows(*rows):
```

Comment:
Humm, agree it's not clean, and I thought about this, but:
- calling `@session` decorator only once every `chunk` instead of once every row is one of the optimization of the PR
- wrapping one `chunk` in a DB transaction is another optimization
- if I factorize those two pieces of code, it means moving those logics to the `helpers` module, and I'm not sure we want this module to depend on the `db` module, plus I'm not sure the session decorator should be integrated in the `batch` stack.

Anyway, I'll have a new look at it.